types/web3/web3-tests.ts

```diff
@@ -10,8 +11,25 @@ const contractAddress = "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe";
 // --------------------------------------------------------------------------
 const web3 = new Web3();
 const myProvider = new web3.providers.HttpProvider("http://localhost:5454");
-web3.setProvider(myProvider);
-web3.eth.setProvider(myProvider);
```

Comment:
Dont remove this.

Reply:
Ah you're right. That's my bad, I just fixed it.